utils.zsh/20-host.zsh

```diff
@@ -56,7 +56,7 @@ _setup_macos() {
 }
 
 _setup_linux() {
-  autoload -Uz is-at-least check_deps check_git check_nasm check_cross
+  autoload -Uz is-at-least check_deps check_git check_nasm check_cross check_cross_windows_arm64
```

Comment:
Rename `check_cross` to `check_cross_windows_x86` to make it clear it's separated by architecture.

Reply:
done